scheds/c/scx_central.bpf.c

```diff
@@ -151,7 +151,7 @@ static bool dispatch_to_cpu(s32 cpu)
 		 * If we can't run the task at the top, do the dumb thing and
 		 * bounce it to the fallback dsq.
 		 */
-		if (!bpf_cpumask_test_cpu(cpu, p->cpus_ptr)) {
+		if (!bpf_cpumask_test_cpu(cpu, p->cpus_ptr) || is_migration_disabled(p)) {
```

Comment:
Since we're in `ops.dispatch()` I think we need to check `p->migration_disabled` directly here. Can you try this instead of using `is_migration_disabled()`?